js/modules/migrations/get_placeholder_migrations.js

```diff
@@ -0,0 +1,23 @@
+const Migrations0DatabaseWithAttachmentData =
+  require('./migrations_0_database_with_attachment_data');
+const Migrations1DatabaseWithoutAttachmentData =
+  require('./migrations_1_database_without_attachment_data');
+
+
+exports.getPlaceholderMigrations = () => {
+  const last0MigrationVersion =
+    Migrations0DatabaseWithAttachmentData.getLatestVersion();
+  const last1MigrationVersion =
+    Migrations1DatabaseWithoutAttachmentData.getLatestVersion();
+
+  const lastMigrationVersion = last1MigrationVersion || last0MigrationVersion;
+
+  return [{
+    version: lastMigrationVersion,
+    migrate() {
+      throw new Error('Unexpected invocation of placeholder migration!' +
```

Comment:
Nice backstop. 👍

Reply:
Thanks! This his how I found our tests were failing 😉 

I think this will be a big win towards moving away from Backbone IndexedDB adapter as this PR does away with the dependency on its migration mechanism in regular app code. We could replace the existing migrations on startup with vanilla IDB code and the rest of the code would continue to work 😄